superset/db_engine_specs/ocient.py

```diff
@@ -23,10 +23,13 @@
 from pyocient import _STPoint, _STLinestring, _STPolygon
 from ipaddress import IPv4Address, IPv6Address
 from superset import app
+from superset.models.sql_lab import Query
 
 superset_log_level = app.config['LOG_LEVEL']
 pyocient.logger.setLevel(superset_log_level)
 
+# Store mapping of query -> cursor object in order to cancel
+cursor_cache = {}
```

Comment:
This deserves a comment describing what is going on here at a high level

Reply:
added